### PR TITLE
Update requirements.txt to regulate taichi version

### PR DESCRIPTION
**Pull request by deehrlic**
Current version of taichi is 1.2.1 - running with that version causes mask 3D demo at minimum to break, using 1.0.3 like in the README fixed it, there must have been some syntax changes in the meantime

### CODE REVIEW

**Comment by deehrlic** [comment]
also added data/fits/ folder with a dummy inside to make sure folder still exists. When you export without a fits folder you get an I/O error!

File: data/fits/fits.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Polyphy may crash if this folder doesn't exist!
```
Comment (PatriceJada) [inline data/fits/fits.txt:1]: The best way to handle it is to use Python to check if the path exists else create it.

**Review by PatriceJada — COMMENTED** [review]
@deehrlic thank you for the contribution. Could you amend your commits to one.

File: data/fits/fits.txt (hunk 1 above)
Comment (deehrlic) [reply data/fits/fits.txt]: ive squashed the commits and made this fix for all the jupyter notebooks + removed fits folder - let me know if there's anything else I can do here, thanks!

Comment (PatriceJada) [reply data/fits/fits.txt]: Looks good to me. Thank you for the addition @deehrlic .

**Pull request merged**